<?xml version="1.0"?>
<launch>
    <arg name="topic" default="/webcam/image_raw" />
    <arg name="show_image_frame" default="true" />
    <arg name="cascade_file" default="$(find face_detection)/include/cascade.xml" />
    <node pkg="eulerian_motion_magnification" type="pulse_measure.py" name="eulerian_motion_magnification" output="screen">
        <param name="topic" type="string" value="$(arg topic)" />
        <param name="show_image_frame" type="bool" value="$(arg show_image_frame)" />
        <param name="cascade_file" type="string" value="$(arg cascade_file)" />
    </node>
</launch>


